sonar-plugin-api/src/main/java/org/sonar/api/rules/RuleFinder.java

```diff
@@ -30,7 +30,7 @@
 
 /**
  * @since 2.3
- * @deprecated since 5.1 DB access will soon be removed from batch side. Use {@link ActiveRules} instead.
+ * @deprecated since 5.1. Use {@link ActiveRules} on batch side.
  */
 @Deprecated
 public interface RuleFinder extends TaskComponent, ServerComponent {
```

Comment:
![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)